docs/coding-conventions.md

```diff
@@ -37,7 +37,30 @@ import os
 # x = "ok"
 ```
 
-### CC3 ...
+### CC3: Naming fixtures that return `Callable`s
+
+- Use verb as prefix

```

Comment:
I would put this line below the next one. It's a bit confusing when reading

Reply:
@GitHK I am not sure what you mean. Can you please provide an edit as before? that would be easier